packages/components/tests/integration/components/hds/table/th-sort-test.js

```diff
@@ -14,31 +14,44 @@ module('Integration | Component | hds/table/th-sort', function (hooks) {
 
   test('it renders', async function (assert) {
     await render(hbs`<Hds::Table::ThSort
-  @sortBy='artist'
-  @sortOrder='asc'
   @setSortBy={{this.setSortBy}}
 >artist</Hds::Table::ThSort>`);
 
     assert.dom(this.element).hasText('artist');
   });
+  test('it has the scope attribute, and it is set to column', async function (assert) {
+    await render(hbs`<Hds::Table::ThSort
+  @setSortBy={{this.setSortBy}}
+>artist</Hds::Table::ThSort>`);
+
+    assert.dom('[data-test-table-th-sort]').hasAttribute('scope', 'col');
+  });
 
   test('it should render the component with a CSS class that matches the component name', async function (assert) {
     await render(hbs`<Hds::Table::ThSort
-  @sortBy='artist'
-  @sortOrder='asc'
   @setSortBy={{this.setSortBy}}
 >artist</Hds::Table::ThSort>`);
 
     assert.dom('[data-test-table-th-sort]').hasClass('hds-table__th-sort');
   });
+
```

Comment:
I would add these tests
- tests the `onClick` functionality of the button, to check that invokes the callback provided with the argument passed to it
- test that the `aria-sort` attribute is correcly set

Reply:
added

Reply:
Sorry, but I don't see them? 🤔